README.md

```diff
@@ -1,133 +1,111 @@
-# Ankermake M5 Protocol
+# AnkerMake M5 Protocol
 
-Welcome! This repository contains `ankerctl.py`, a command-line interface for
-monitoring, controlling and interfacing with Ankermake M5 3D printers.
+Welcome! This repository contains `ankerctl.py`, a command-line interface for monitoring, controlling and interfacing with AnkerMake M5 3D printers.
 
-NOTE: This software is in early stages, so expect sharp edges and occasional errors!
+**NOTE:** This software is in early stages, so expect sharp edges and occasional errors!
 
-The `ankerctl` program uses [`libflagship`](documentation/libflagship.md), a
-library for communicating with the numerous different protocols required for
-connecting to an Ankermake M5 printer. The `libflagship` library is also maintained
-in this repo, under [`libflagship/`](libflagship/).
+The `ankerctl` program uses [`libflagship`](documentation/libflagship.md), a library for communicating with the numerous different protocols required for connecting to an AnkerMake M5 printer. The `libflagship` library is also maintained in this repo, under [`libflagship/`](libflagship/).
 
-## Current features
+## Features
 
- - Print directly from PrusaSlicer, SuperSlicer
+### Current Features
 
- - Connect to Ankermake M5 and Ankermake APIs without using closed-source Anker
-   software.
+ - Print directly from PrusaSlicer and its derivatives (SuperSlicer, Bamboo Studio, OrcaSlicer, etc.)
 
- - Send raw gcode commands directly to the printer (and see the response)
+ - Connect to AnkerMake M5 and AnkerMake APIs without using closed-source Anker software.
 
- - Low-level access to MQTT, PPPP and HTTPS APIs
+ - Send raw gcode commands directly to the printer (and see the response).
 
- - Send print jobs to the printer
+ - Low-level access to MQTT, PPPP and HTTPS APIs.
 
- - Stream camera image/video to your computer
+ - Send print jobs (gcode files) to the printer.
 
+ - Stream camera image/video to your computer.
 
-## Upcoming and planned features
+ - Easily monitor print status.
 
- - Easily monitor print status
+### Upcoming and Planned Features
 
- - Integration into other software. Home Assistant? Prusa Slicer?
+ - Integration into other software. Home Assistant? Cura plugin?
 
-Pull requests always welcome :-)
+Let us know what you want to see; Pull requests always welcome! :smile:
 
-## Installation instructions
+## Installation (Prerequisite to the Usage section)
```

Comment:
I understand what you're saying here, but maybe the phrase "Prerequisite to the Usage section" isn't the easiest for non-native speakers?

How about we refer to the installation instructions in the usage section, instead?

Reply:
I would posit that if we just called it Installation that individuals would understand they have to do that to be able to use it. For now unless you object I renamed it "Installation"